### DISCUSSION TITLE
How to use journals

### DISCUSSION THREAD

**Discussion by wehkah** [Kozea/Radicale#1481] [Q&A general]
Hello,

I am looking for information on how to use journals with Radicale2. The program offers different kind of calendars, one of which includes the option "journal", but I don't know how to use this feature. The Documentation on radicale.org/ doesn't mention this issue either.

Could some kind soul please tell me what programs/apps I could use for this feature on my Android device and on my Linux machine? 

Thanks in advance!

**Comment by anlar** [comment]
Type of calendar specifies which of iCal objects it supports. "Journal" means VJOURNAL objects, see https://en.wikipedia.org/wiki/ICalendar#Journal_entry_(VJOURNAL).

**Comment by dstrelnikov** [comment]
Haven't tried it myself, but here is one Android app for journals: https://jtx.techbee.at/.

**Comment by tgf9** [comment]
Any Linux desktop app recommendations to work with journals?

**Reply by wehkah** [reply]
Did you find a Linux app yet? I'd be interested in that, too.

**Reply by leonid-chaichenets** [reply]
I believe [Gnome’s Evolution](https://gitlab.gnome.org/GNOME/evolution) can work with notes based on VJOURNAL.
